docs/error.md

```diff
@@ -131,6 +131,21 @@ There should be an earlier warning giving more information why type `T` is not s
 
 Note: Supported features will evolve with new versions of the tool.
 
+<h3><a name="EM1041"/>Indexed properties on `T` is not generated because multiple indexed properties are not supported.</h3>
+
+This is a **warning** that the indexed properties on `T` will be ignored (i.e. nothing will be generated) because multiple indexed properties are not supported.
+
+Note: Supported features will evolve with new versions of the tool.
```

Comment:
I don't think we can support it (in the future) right ?

In this case we should say **it cannot be supported** and remove the `Note` part.